input/pagecontent/index.md

```diff
@@ -27,7 +27,7 @@ This guide is divided into several pages which are listed at the top of each pag
   - [AU eRequesting data for interoperability](auereqdi.html): This is a placeholder for when content becomes available.
   - [Relationship with other HL7 AU FHIR IGs](relationship.html): This is a placeholder for when content becomes available.
   - [AU Variance Statement](variance.html): This page documents variance from AU Base and AU Core.
-  - [Future of AU eRequesting](erequesting-future.html): This is a placeholder for when content becomes available.
+  - [Future of AU eRequesting](future.html): This is a placeholder for when content becomes available.
```

Comment:
Suggest removing placeholder and updating in line with Core: "This page outlines the approach to developing AU eRequesting and yearly update cycle."

Reply:
changed.